lib/utils.js

```diff
@@ -38,7 +38,7 @@ function hasCEReactions(idl) {
 }
 
 function isOnInstance(memberIDL, interfaceIDL) {
-  return memberIDL.special !== "static" && (getExtAttr(memberIDL.extAttrs, "Unforgeable") || isGlobal(interfaceIDL));
+  return memberIDL.special !== "static" && (getExtAttr(memberIDL.extAttrs, "LegacyUnforgeable") || isGlobal(interfaceIDL));
```

Comment:
This line is too long, which is why the build is failing:

```suggestion
  return (
    memberIDL.special !== "static" &&
    (getExtAttr(memberIDL.extAttrs, "LegacyUnforgeable") || isGlobal(interfaceIDL))
  );
```

The above is output from **Prettier**.